biomage/utils/data.py

```diff
@@ -84,13 +88,29 @@ def modify_records(item, target_table, config, **extra):
         for samples_id in item["projects"]["M"]["samples"]["L"]:
             new_samples_list.append({"S" : f"{extra['sandbox_id']}-{samples_id['S']}"})
 
-        item["projects"]["M"]["experiments"]["L"] = new_samples_list
+        item["projects"]["M"]["samples"]["L"] = new_samples_list
 
         return item
 
     return {}
 
 
+def prefix_cell_set_samples_key(sandbox_id, cell_sets):
+    """
+    Prefix sandbox_id to sample keys in cell-sets object
+    """
```

Comment:
Can you add a comment explaining why this prefixing is needed?

Reply:
This is no longer needed as all prefix to samples are removed. More reason stated in PR description.